XpringKit/JavaScript/JavaScriptUtils.swift

```diff
@@ -6,27 +6,32 @@ internal class JavaScriptUtils {
 	/// String constants which refer to named JavaScript resources.
 	private enum ResourceNames {
     public static let address = "address"
-    public static let tag = "tag"
-    public static let isValidAddress = "isValidAddress"
     public static let decodeXAddress = "decodeXAddress"
     public static let encodeXAddress = "encodeXAddress"
+    public static let isValidAddress = "isValidAddress"
+    public static let isValidClassicAddress = "isValidClassicAddress"
+    public static let isValidXAddress = "isValidXAddress"
+    public static let tag = "tag"
     public static let utils = "Utils"
 	}
 
 	/// Native javaScript functions wrapped by this class.
-	private let isValidAddressFunction: JSValue
-    private let encodeXAddressFunction: JSValue
-    private let decodeXAddressFunction: JSValue
+  private let encodeXAddressFunction: JSValue
```

Comment:
indentation looks weird? or maybe it's just the way github is showing it

Reply:
Sorry about that.  Fixed. 

I think that at some point I had xcode set to tabs vs spaces and it messed up the formatting. Outside of this CL, I'll look at whether we can catch that with `swiftlint` or if I can do a global file reformat.